ProtoWorkspace/ProjectFileInfo.fs

```diff
@@ -51,150 +52,131 @@ type OutputType =
     | Library
     /// Build a module that can be added to another assembly (.netmodule)
     | Module
-
-    override self.ToString () = self |> function
-        | Exe     -> Constants.Exe
-        | Winexe  -> Constants.Winexe
+    
+    override self.ToString() = 
+        self |> function 
+        | Exe -> Constants.Exe
+        | Winexe -> Constants.Winexe
         | Library -> Constants.Library
-        | Module  -> Constants.Module
-
-    static member Parse text = text |> function
-        | EqualsIC Constants.Exe     -> Exe
-        | EqualsIC Constants.Winexe  -> Winexe
+        | Module -> Constants.Module
+    
+    static member Parse text = 
+        text |> function 
+        | EqualsIC Constants.Exe -> Exe
+        | EqualsIC Constants.Winexe -> Winexe
         | EqualsIC Constants.Library -> Library
-        | EqualsIC Constants.Module  -> Module
-        | _ ->
-            failwithf "Could not parse '%s' into a `OutputType`" text
-
-    static member TryParse text = text |> function
-        | EqualsIC Constants.Exe     -> Some Exe
-        | EqualsIC Constants.Winexe  -> Some Winexe
+        | EqualsIC Constants.Module -> Module
+        | _ -> failwithf "Could not parse '%s' into a `OutputType`" text
+    
+    static member TryParse text = 
+        text |> function 
```

Comment:
it crams a bunch of stuff onto some lines, but this one needs to be on a separate line? absurd